src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/environment/RequiredEnvironmentVariables.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.environment;
+
+public enum RequiredEnvironmentVariables {
+    
+    ITEMS_DATABASE("ITEMS_DATABASE"),
+    MONGODB_URL("MONGODB_URL");
+    
```

Comment:
Don't we need to treat the variables used when talking to other APIs required as well? See https://github.com/companieshouse/certified-copies.orders.api.ch.gov.uk/blob/b56b50020caa700b3adcea75cb74d6d7a14ace2f/src/test/java/uk/gov/companieshouse/certifiedcopies/orders/api/util/TestUtils.java#L17-#L19 for instance.

Reply:
I added the variables declared in the env file for the application. Happy to add the other variables if they are being declared for this application.
Do we have a list of all of the applicable variables to check?

Reply:
It seems some of these variables are actually defined in places such as `global_env`, so it may be unlikely that they will not be set. Nonetheless they are effectively required because the app will not work correctly without them.
We do not have a list of required variables, but in the [equivalent PR I raised recently](https://github.com/companieshouse/certificates.orders.api.ch.gov.uk/pull/58/files), I did add descriptions of the extra required variables to the `README`.
We might like to consider some kind of environment variables registry in Confluence or somewhere going forward? Alternatively that could be covered by a best practice of always updating a release note whenever a dependency on an environment variable is introduced?

Reply:
I've added the 3 environment variables as suggested by @VaughanJackson123

Reply:
Was the outcome of discussions (in stand up?) last week that we should document the variables in the `README` too?

Reply:
I've added the end points and variables to the README

Reply:
I also noticed a number of tabs in the test class file so formatted the file to remove them and unused imports